noir-projects/aztec-nr/aztec/src/authwit/auth_selector.nr

```diff
@@ -4,32 +4,32 @@ use dep::protocol_types::{
 };
 
 #[derive(Eq, Serialize, Deserialize)]
-pub struct AuthwitSelector {
+pub struct AuthorizationSelector {
     // 1st 4-bytes (big-endian leftmost) of abi-encoding of an authwit.
```

Comment:
This doesn't seem to be 4 bytes and the TODO below doesn't apply

Reply:
Actually reverted, seems like we keep the full field but just use 4 bytes on TS? Constraint saving measure?